ext/opcache/jit/README.md

```diff
@@ -24,5 +24,119 @@ was almost 100 times slower, making it prohibitively expensive to use.
 [The unofficial DynASM Documentation](https://corsix.github.io/dynasm-doc/tutorial.html)
 has a tutorial, reference, and instruction listing.
 
-`zend_jit_x86.dasc` gets automatically converted to `zend_jit_x86.c` by the bundled
+In x86 builds, `zend_jit_x86.dasc` gets automatically converted to `zend_jit_x86.c` by the bundled
 `dynasm` during `make`.
+
+In arm64 builds, `zend_jit_arm64.dasc` gets automatically converted to `zend_jit_arm64.c` by the bundled
+`dynasm` during `make`.
+
+Running tests of the JIT
+------------------------
+
+Then, to test the JIT, e.g. with opcache.jit=tracing, an example command
+based on what is used to test in Azure CI:
+
+```
+make test TESTS="-d opcache.jit_buffer_size=16M -d opcache.enable=1 -d opcache.enable_cli=1 -d opcache.protect_memory=1 -d opcache.jit=tracing --repeat 2 --show-diff -j$(nproc) ext/opcache Zend"
+```
+
+- `opcache.jit_buffer_size=16M` enables the JIT in tests by providing 16 megabytes of
+  memory to use with the JIT to test with.
+- `opcache.protect_memory=1` will detect writing to memory that is meant to be
+  read-only, which is sometimes the cause of opcache bugs.
+- `--repeat 2` is optional, but used in CI since some JIT bugs only show up after processing a
+  request multiple times (the first request compiles the trace and the second executes it)
+- `-j$(nproc)` runs as many workers to run tests as there are CPUs.
+- `ext/opcache/` and `Zend` are the folders with the tests to run, in this case opcache
+  and the Zend engine itself.  If no folders are provided, all tests are run.
+
+When investigating test failures such as segmentation faults,
+configuring the build of php with `--enable-address-sanitizer` to enable
+[AddressSanitizer](https://github.com/google/sanitizers/wiki/AddressSanitizer) is often useful.
+
+Some of the time, adding `-m --show-mem` to the `TESTS` configuration is also useful to test with [valgrind](https://valgrind.org/) to detect out of bounds memory accesses.
+This is less effective at detecting or explaining invalid memory accesses than AddressSanitizer, but does not require rebuilding php.
```

Comment:
Not really less effective, just much slower. You can rebuild PHP in the time it takes to run a handful of tests under valgrind ;)

Reply:
https://github.com/google/sanitizers/wiki/AddressSanitizerAlgorithm#short-version

I mean that I thought AddressSanitizer would detect the computation of an invalid address, which might be distanced from the **use** (read/write) of the invalid address that caused misbehaviors.

Reply:
No, address sanitizer does not detect invalid address calculations. Though I believe ubsan does, at least to a limited degree.

Reply:
Oh. Missed the `*` when reading that.